package.json

```diff
@@ -41,19 +41,19 @@
     "request-promise": "^0.x"
   },
   "devDependencies": {
-    "buddy.js": "^0.x",
```

Comment:
This is a consequence of the move to node-semver@^3.0.0, which changes the interpretation of the ^ operator for 0.x.y versions. The short version is that for versions < 1.0.0, ^ now works the same as =. If you want the old behavior, use ~ instead of ^. This is one of the breaking changes in npm@2.0.0. It's an irritating change, but this is how semver.org mandates that 0.x.y versions be dealt with.